test/e2e_env/gatewayapi/conformance_test.go

```diff
@@ -106,8 +106,8 @@ func TestConformance(t *testing.T) {
 		RestConfig:           clientConfig,
 		Clientset:            clientset,
 		GatewayClassName:     "kuma",
-		CleanupBaseResources: true,
-		Debug:                Config.Debug,
+		CleanupBaseResources: !Config.Debug, // we need to keep the resources to collect logs when Debug is enabled
```

Comment:
Note that this each test case will cleanup its own resources no matter what, so don't expect too much from this